mender/install_mender.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+
+
+function get_mender_connect()
+{
+   [ ! -d ./mender/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender/mender-connect
```

Comment:
Following the previous suggestions 
https://github.com/perimeter-inc/system-dependencies/pull/4/files#r808020982
```suggestion
   [ ! -d ${SCRIPT_PATH}/mender/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ${SCRIPT_PATH}/mender/mender-connect
```